src/land/ghcn_snod2ioda.py

```diff
@@ -180,8 +178,7 @@ def assignValue(colrowValue, df400):
         self.outdata[self.varDict[iodavar]['errKey']] = errs
         self.outdata[self.varDict[iodavar]['qcKey']] = qflg
 
-        DimDict['nlocs'] = len(self.outdata[('datetime', 'MetaData')])
-        AttrData['nlocs'] = np.int32(DimDict['nlocs'])
+        DimDict['Location'] = len(self.outdata[('datetime', 'MetaData')])
```

Comment:
I think these are now supposed to be `dateTime`. May require my PR to be merged in before it will fully work.

Reply:
Corrected.